cargo-dist/src/lib.rs

```diff
@@ -966,7 +966,26 @@ impl Library {
         if let Some(prefix) = brew_prefix {
             let cloned = library.clone();
             let stripped = cloned.strip_prefix(prefix).unwrap();
-            let package = stripped.split('/').nth(0).unwrap();
+            let mut package = stripped.split('/').nth(0).unwrap().to_owned();
+
+            // The path alone isn't enough to determine the tap the formula
+            // came from. If the install receipt exists, we can use it to
+            // get the name of the source tap.
+            let receipt = Utf8PathBuf::from(&prefix)
+                .join(&package)
+                .join("INSTALL_RECEIPT.json");
+
+            if receipt.exists() {
+                let _ = SourceFile::load_local(&receipt).and_then(|file| {
```

Comment:
just checking we are in fact intentionally ignoring all errors and silently moving forward?

Reply:
Added a comment to that effect.